src/COMPONENTS/PRODUCTSDETAILS/productdetails.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
+import React, { useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { fetchProducts } from "../../REDUX/products/productsSlice";
+import { getProductDetails } from "../../REDUX/products/productsdetailsSlice";
+import "./productdetails.css";
+
+const ProductDetails = () => {
+  const { id } = useParams();
+  const { product } = useSelector((state) => state.productDetail);
+  const dispatch = useDispatch();
+  
+  useEffect(() => {
+    dispatch(fetchProducts())
+      .unwrap()
+      .then((result => {
+        dispatch(getProductDetails({ products: result, id: Number(id) }));
+      }));
+  }, []);
+  return (
+    <div id={product.id}>
+      <Link to="/">
+        <button className="goback-btn"> Click here to go back</button>
+      </Link>
```

Comment:
- [ ] For the back button Please add it to the header component because it is a part of the navbar and kindly replace `Click here to go back` with `<` to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/189375519-d58c1241-48d7-4bc9-a64a-2873a47a92b7.png) | ![image](https://user-images.githubusercontent.com/68796232/189375495-149affb9-ea3e-4a43-b2a4-9cb0d4f8f2d1.png) |

[Hint] You can add it to the header component using conditional rendering.

Reply:
thank you @Omar-Muhamad , change made